lib/authorization.rb

```diff
@@ -55,8 +55,10 @@ def self.generate_default_acls(subject, object)
       acls: [
         {subjects: [subject], actions: ["Project:view", "Project:delete", "Project:user", "Project:policy", "Project:billing", "Project:github"], objects: [object]},
         {subjects: [subject], actions: ["Vm:view", "Vm:create", "Vm:delete"], objects: [object]},
+        {subjects: [subject], actions: ["Vm:Firewall:view", "Vm:Firewall:edit"], objects: [object]},
         {subjects: [subject], actions: ["PrivateSubnet:view", "PrivateSubnet:create", "PrivateSubnet:delete", "PrivateSubnet:nic"], objects: [object]},
-        {subjects: [subject], actions: ["Postgres:view", "Postgres:create", "Postgres:delete"], objects: [object]}
+        {subjects: [subject], actions: ["Postgres:view", "Postgres:create", "Postgres:delete"], objects: [object]},
+        {subjects: [subject], actions: ["Postgres:Firewall:view", "Postgres:Firewall:edit"], objects: [object]}
```

Comment:
We should probably backfill these permissions to the appropriate users like we did in Postgres Launch.

Also, I think we badly need * permissions. I wish I added that to gridding :smile:. Maybe we could squeze it in if we find some time.

Reply:
Actually, I developed a prototype during the PGConf.EU event. It could potentially be ready for merging within a day.

https://github.com/ubicloud/ubicloud/commit/63b9639b4d346b8531e3ced252128b19e5fe7662

Reply:
We will handle it together with the `Vm:Firewall:` permissions before the deployment